modules/ROOT/pages/view-api-contracts.adoc

```diff
@@ -0,0 +1,27 @@
+= View API Instance Contracts 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: view contracts, api, contracts, viewing
+
+From API Manager, you can view the contracts created for your API instance when you grant access to that instance.
+
+To view your API instance contracts:
+
+. In API Manager, click *API Administration*:
++
+image::api-manager-administrator.png[api-manager-administrator]
++
+. Under the name of the API, click *>* to expand the API instance list.
+. Click the `<api version>:<api instance>` link, for example, `v1:16046365`. 
```

Comment:
Same as above. If the API Instance has a Label, then the label is shown.

Reply:
Done! Thanks!